docs/section6/Rule6-13.md

```diff
@@ -10,41 +10,69 @@
 - Table G3.7, Performance Rating Method Lighting Power Density Allowances and Occupancy Sensor Reductions Using the Space-by-Space Method  
 
 **Applicability:** All required data elements exist for P_RMR  
-**Applicability Checks:** None  
+**Applicability Checks:**  
+
+  1. Pass Rule 6-10.
+
```

Comment:
not sure why we need the applicability check for 6-10. 6-10 checks for occupancy controls in the baseline. occupancy controls in proposed are required irrespective of the outcome of 6-10. @mkarpman - any thoughts?

Reply:
Yes, I agree.

Reply:
Applicability check is now removed.